src/node/nodestate.h

```diff
@@ -871,7 +869,7 @@ namespace ccf
 
     std::optional<std::vector<uint8_t>> get_quote()
     {
-      std::vector<uint8_t> quote{1};
+      std::vector<uint8_t> quote;
```

Comment:
This shadows the member variable now named `quote`. This will work until it doesn't. Recommend naming the local variable `q` or similar.